src/main/java/org/zaproxy/zap/extension/jwt/JWTActiveScanRule.java

```diff
@@ -105,9 +105,7 @@ public void scan(HttpMessage msg, String param, String value) {
         }
 
         if (JWTConfiguration.getInstance().isEnableClientConfigurationScan()) {
-            if (performAttackClientSideConfigurations(msg, param)) {
-                return;
-            }
```

Comment:
@kingthorin @thc202 i think there is no need to stop the addon on finding any UI related vulnerability.

Reply:
Makes sense.